server/container/util/src/main/java/org/apache/james/util/CompletableFutureUtil.java

```diff
@@ -101,6 +102,12 @@
                 stream.map(action));
     }
 
+    public static <T> CompletableFuture<Stream<T>> sort(CompletableFuture<Stream<T>> futurStream, Comparator<T> comparator) {
+        return futurStream
+            .thenApply(stream ->
+                stream.sorted(comparator));
```

Comment:
I don't get your point? When I have a`CompletableFuture<Stream>`, I need to sort elements inside the stream. Rather than using `CompletableFuture.thenApply()` then `Stream.sorted()`. I would like to use the shorter api from `CompletableFutureUtil`. That is my point :)